build.psm1

```diff
@@ -2114,6 +2116,27 @@ function Start-CrossGen {
     }
 }
 
+# Cleans the PowerShell repo
+# by default everything but the root folder and the Packages folder
+# if you specify -IncludePackages it will clean the Packages folder
+function Start-PSCleanRepo
+{
+    [CmdletBinding()]
+    param(
+        [switch] $IncludePackages
+    )
+    Get-ChildItem $RepoRoot\* -Directory -Exclude 'Packages' | ForEach-Object {
```

Comment:
I don't see `$RepoRoot` is defined anywhere in build.psm1. Am I missing something?